rnd/autogpt_builder/src/components/node-input-components.tsx

```diff
@@ -259,29 +282,37 @@ const NodeKeyValueInput: FC<{
   schema: BlockIOKVSubSchema;
   entries?: { [key: string]: string } | { [key: string]: number };
   errors: { [key: string]: string | undefined };
+  connections: NodeObjectInputTreeProps["connections"];
   handleInputChange: NodeObjectInputTreeProps["handleInputChange"];
   className?: string;
   displayName?: string;
 }> = ({
   selfKey,
   entries,
   schema,
+  connections,
   handleInputChange,
   errors,
   className,
   displayName,
 }) => {
+  let defaultEntries = new Map<string, any>();
```

Comment:
fields for key-value pair, can be defined from:
1. connected pin
2. hard-coded default values